machine-learning/Intro To Machine Learning.ipynb

```diff
@@ -0,0 +1,320 @@
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "       public class HouseData\n",
+        "       {\n",
+        "           public float Size { get; set; }\n",
+        "           public float Price { get; set; }\n",
+        "       }"
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Then define the house price prediction data structure"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "       public class Prediction\n",
+        "       {\n",
+        "           [ColumnName(\"Score\")]\n",
+        "           public float Price { get; set; }\n",
+        "       }"
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Now we are ready to train the pre-collected data we'll use for the house price prediction scenario"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "           HouseData[] houseData = {\n",
+        "               new HouseData() { Size = 1.1F, Price = 1.2F },\n",
+        "               new HouseData() { Size = 1.9F, Price = 2.3F },\n",
+        "               new HouseData() { Size = 2.8F, Price = 3.0F },\n",
+        "               new HouseData() { Size = 3.4F, Price = 3.7F } };"
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Using the `MLContext` we previously created, load the training data into ML.NET [IDataView](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.idataview?view=ml-dotnet) which is the fundamental ML.NET data pipeline type"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "IDataView trainingData = mlContext.Data.LoadFromEnumerable(houseData);"
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Now we have the data ready, next we'll create the ML.NET pipeline specifying the training model we are going to use for our ML operation. For house price prediction, we are going to use the Regression transformer model. ML.NET supports other machine learning models which can be used for other scenarios as needed. The pipeline will create what is called [Estimator](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.iestimator-1?view=ml-dotnet) which used for training the data"
```

Comment:
Removed localization. Also reworder a few things. This would also be a good opportunity to discuss what a trainer is. 

```suggestion
        "Now we have the data ready, next we'll create the ML.NET pipeline specifying the trainer we are going to use to build our machine learning model. For house price prediction, we are going to use the regression trainer. ML.NET supports other machine learning trainers which can be used for other scenarios as needed. The pipeline will create what is called [Estimator](https://docs.microsoft.com/dotnet/api/microsoft.ml.iestimator-1?view=ml-dotnet) which used to define the operations applied to the data."
```